node/src/node/slave/machine.cpp

```diff
@@ -237,6 +244,8 @@ auto machine_t::inject(load_t& load, channel_handler handler) -> std::uint64_t {
         });
     }
 
+    metrics_data.load->add(current);
```

Comment:
Why adding value is so far from retriving it? Looks like it can be racy and values can be swapped:
  * obtain current load (1)
  * decrease load, register decreased load(0)
  * register increased load(1), while real load is 0
  * wait a little and get biased metric